tox.ini

```diff
@@ -23,27 +26,33 @@ platform =
     windows: win32
 usedevelop = True
 setenv =
-    {py36,py37,py38,py39,py310}-{linux,darwin,windows}: COVERAGE_FILE = .coverage.{envname}
+    COVERAGE_FILE = .coverage.{envname}
 commands =
     {envpython} -m pytest --cov --cov-report=term-missing:skip-covered {posargs}
 depends =
-    {py36,py37,py38,py39,py310}-{linux,darwin,windows}: clean
-    report: {py36,py37,py38,py39,py310}-{linux,darwin,windows}
+    clean
 
 [testenv:clean]
 deps = coverage
 skip_install = true
+setenv =
+    COVERAGE_FILE =
 commands = coverage erase
+depends =
 
 [testenv:report]
 deps = coverage
 skip_install = true
 parallel_show_output = true
+setenv =
+    COVERAGE_FILE =
 commands =
     coverage combine
     coverage report --show-missing
     coverage html
     coverage xml
+depends =
+    py3{7,8,9,10,11}-{linux,darwin,windows}
```

Comment:
Unfortunately, the repetition seems unavoidable, but at least it can be copied/pasted as a whole line now.

Reply:
No problem - thanks for looking into this. This is definitely an improvement.